.github/scripts/Dockerfile.builder

```diff
@@ -0,0 +1,3 @@
+FROM fedora:34
```

Comment:
Docker (and also Podman) have support for multistage Dockerfiles. https://docs.docker.com/develop/develop-images/multistage-build/ This approach would be imo cleaner and more obvious.